hypervisor/src/hypervisor/config.rs

```diff
@@ -1,3 +1,52 @@
+//! Configuration for a653rs-linux-hypervisor.
+//!
+//! A configuration contains information about the partition schedule.
+//! Currently, only a single schedule is supported. Each schedule's major frame
+//! (MAF) has a fixed duration and number of slots. Each slot is occupied by
+//! at most one partition that executes during the slot. A slot has a fixed
+//! duration and offset inside the MAF. A partition may occupy multiple slots
+//! inside the schedule, in which case it may be repeated using the `period`
+//! parameter.
```

Comment:
Why are you doing line-breaks manually

Reply:
What would be the alternative? 80 characters seems to be the default. I'd like to keep the documentation readable, regardless of the editor. Looked here for reference on how to do this. https://doc.rust-lang.org/src/std/lib.rs.html#1-686

Reply:
acceptable